entry.py

```diff
@@ -1,7 +1,34 @@
-from autologging import logged, traced
+import logging, sys
+from typing import Any, Union
+
+from autologging import logged
+from exam_date.stored_date import AssignmentLatestSubmittedDate
+from spe_utils import utils
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv(dotenv_path=os.path.dirname(os.path.abspath(__file__)) + "/.env")
+
+
+@logged
+def set_up_logging():
+    level = utils.LOGGING_LEVEL
+    log_level = os.getenv(level)
+    if log_level:
+        logging.basicConfig(level=log_level, stream=sys.stdout,
+                            format="%(asctime)s - %(levelname)s -  %(name)s - %(funcName)s - %(message)s")
```

Comment:
Is `asctime` in UTC?

Reply:
No, when I ran the script i see the date that local time in 24 hour format

Reply:
Change it to use UTC.

Reply:
I think it's more appropriate for the log timestamp to be in localtime.

Reply:
I feel the same since this is cron script that will be running on a schedule, I feel have the local time is more appropriate

Reply:
🤷‍♂️

Reply:
Store dates as UTC, display dates as America/Detroit!

Reply:
What date be stored as UTC? I am confused or I have confused everybody. The date that is stored in `persisted.txt` is in UTC as that is the date I will be using to query canvas. But the display date for logs I prefer local time. is that what you mean?

Reply:
I was just stating a general principle. This is correct, persisted.txt should contain UTC and log should be local. (So you won't have any issues when the time zone changes)